<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registro</class>
 <widget class="QWidget" name="Registro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registro de estudiantes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTabWidget" name="LecturaTab">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="RegistroTab">
      <attribute name="title">
       <string>Registro</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="8" column="0" colspan="5">
        <widget class="QPushButton" name="AceptarPB">
         <property name="text">
          <string>Aceptar</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QPushButton" name="MostrarPB">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Mostrar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QLineEdit" name="NombreLE">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="2">
        <widget class="QLineEdit" name="MateriasLE"/>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QLineEdit" name="CorreoLE"/>
       </item>
       <item row="4" column="2" colspan="2">
        <widget class="QLineEdit" name="ContrasenaLE">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="NombreL">
         <property name="text">
          <string>Nombre</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="CorreoL">
         <property name="text">
          <string>Correo</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="ContasenaL">
         <property name="text">
          <string>Contraseña</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="MateriasL">
         <property name="text">
          <string>Materias</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="EditarTab">
      <attribute name="title">
       <string>Editar</string>
      </attribute>
      <widget class="QLabel" name="NombreEditL">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="NombreEditLE">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>591</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="AceptarEditLE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>411</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Campos a editar</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>411</width>
         <height>119</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="1">
         <widget class="QLineEdit" name="CorreoEditLE"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="NombreEdit2L">
          <property name="text">
           <string>Nombre</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="CorreoEditL">
          <property name="text">
           <string>Correo</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="MateriasEditL">
          <property name="text">
           <string>Materias</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="MateriasEditLE"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="ContrasenaEditLE"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="ContrasenaEditL">
          <property name="text">
           <string>Contraseña</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="NombreEdit2LE"/>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="NomCB">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="CorrCB">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QCheckBox" name="ContCB">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QCheckBox" name="MateriasCB">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Alumnos</string>
      </attribute>
      <widget class="QPushButton" name="MostrarPB_2">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>0</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Mostrar</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="MensajeTE">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>431</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="EliminarTab">
      <attribute name="title">
       <string>Eliminar</string>
      </attribute>
      <widget class="QLabel" name="NombreElimL">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="NombreElimLE">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QRadioButton" name="ElimTodosRB">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Eliminar todos los alumnos</string>
       </property>
      </widget>
      <widget class="QPushButton" name="AceptarElimPB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>321</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
